_includes/footer.html

```diff
@@ -34,7 +34,7 @@
     {%- endcapture -%}
 
     {%- capture _theme -%}
-      <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a>
+      <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy@{{ theme.version }}</a>
```

Comment:
Drive-by comment, because I really like this idea and I've just been messing about with it locally myself.

How about only displaying it as a tooltip though?
```suggestion
      <span
        data-bs-toggle="tooltip"
        data-bs-placement="top"
        title="Version: {{ theme.version }}"
      >
        <a href="https://github.com/cotes2020/jekyll-theme-chirpy" target="_blank" rel="noopener">Chirpy</a>
      </span>
```

<img width="136" alt="image" src="https://github.com/cotes2020/jekyll-theme-chirpy/assets/12701930/10c9519a-f35d-49e5-94f3-8054270c9345">

Reply:
@GetOutOfMyBakery I vote for your idea.

Reply:
Yes, I like this option as well. In this case the information will be less explicit, but at least it won't be out of the old display format. The question now is which option is better: Implicit or Explicit.

What do you think about shortening it to "v6.5.3" and not using the `span` wrapper?
<img width="274" alt="image" src="https://github.com/cotes2020/jekyll-theme-chirpy/assets/18232567/542bcd52-6b71-45bf-a05a-a38b8443f93f">

Reply:
Implicit and `vX.Y.Z` @MrMurdock11